### ISSUE TITLE
Rich txt toolbar icons missing

### ISSUE THREAD

**Issue by VanguardAX1** [nextcloud/text#1864]
**Describe the bug**
Every icon on the markdown file editing toolbar, except the checkmark is missing.

**To Reproduce**
1. Open any markdown file and edit.
2. Look at the toolbar at top.

**Expected behavior**
A classical full markdown toolbar should be there.

**Screenshots**
<img width="653" alt="missing_icons" src="https://user-images.githubusercontent.com/86813017/134301337-9da70ad6-36f1-49fa-a9c6-e01d58f04c18.PNG">

**Client details:**
 - OS: Windows 10
 - Browser: chrome
 - Version: 22
 - Device: desktop

<details>
<summary>Server details</summary>

**Text app version:** (see Nextcloud apps page)
3.3.0
**Operating system:**
Centos 7
**Web server:**
Apache (httpd)
**Database:**
Mariadb 5
**PHP version:**
7.3.31
**Nextcloud version:** (see Nextcloud admin page)
22.1.1
</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by VanguardAX1** [comment]
update: after some digging it seems [icons.scss file in text app](https://github.com/nextcloud/text/blob/master/css/icons.scss) did not run right for me.
I tried `grep -rl "icon-quote"` and get
```
apps/text/js/editor-rich.js
apps/text/js/editor-rich.js.map
data/appdata_oc8x7z80403e/css/text/593e-5d2e-icons.css
```
There's only `593e-5d2e-icons.css` （and its deps and gzip）under `nextcloud/data/appdata_myInstanceId/css/text` , which reads full of `.icon-undo{background-image:var(--icon-text-undo-000)}.icon-undo-white,.icon-undo.icon-white{background-image:var(--icon-text-undo-fff)}` and so on. Seems those app-icons are not drawn.
Not sure how to fix that. Compressed js/css read like riddle to me.

**Comment by VanguardAX1** [comment]
update: Don't know what happened. I cleared chrome caches and icons showed up.
What I did between:
I installed `php-bcmath php-gmp `  and restarted apache. That does not feels like the cause to me :(

**Comment by juliushaertl** [comment]
Could be a browser cache issue, or maybe the scss cache did not get updated properly. In both cases that would be a server issue, but i cannot really reproduce that.

**Comment by mejo-** [comment]
I would recommend to close this issue as it's been resolved for the submitter and cannot be reproduced/retraced anymore.

**Comment by pierreozoux** [comment]
Ok, I have the same issue:

On a deploy, this file is empty:
`/apps/accessibility/css/user-d41d8cd98f00b204e9800998ecf8427e`

And on the other, this is fine:
`/apps/accessibility/css/user-2386744a4e3332d62b0b59c074e34127`

And I found the route cause.

We are currently investigating a workaround for this issue:
https://github.com/nextcloud/server/issues/32385

We try to make kubernetes restart the pod when it happens.
We are currently using a php-fpm scripts, but there is a bug and it is not enough to restart the pod.
So we decided to go mono pod, with nginx and fpm in the same pod, in 2 different containers.

But as we do have [caching in nginx](https://lab.libreho.st/libre.sh/docker/nextcloud/-/blob/master/web/default.conf.template#L91-92) and now I think the traffic arrives, nginx request the resource to php-fpm, and maybe it is not ready, so then nginx gets a bad response and cache this bad response.

I'll put back the readiness check, and come back to report here if it worked.

But yeah, this definitely looks like a server configuration issue.

I report here to document eventually for other people.
